app/assets/stylesheets/pages/_events_show.scss

```diff
@@ -0,0 +1,55 @@
+
+.logo {
+text-align: center;
+font-weight: 300;
+font-size: 35px;
+}
+
+.title1 {
+  text-align: center;
+  top: 40px;
+  font-weight: bolder;
+  margin-top: 100px;
+  font-size: 30px;
+}
+
+.button1 {
+  vertical-align: center;
+}
+
+.card-center {
+  text-align: -webkit-center;
+}
+
+.small-text {
+  font-size: 15px;
+  text-align: top;
+}
+
+.add {
```

Comment:
what does add mean? Super-undescriptive class name. Please change

Reply:
This is for the add button on the button. I will add button to the css